mxbluesky/devices/zebra.py

```diff
@@ -0,0 +1,700 @@
+print(f"Loading {__file__}")
```

Comment:
Only the initial devices are similar, the actual Zebra device and its parent classes are fairly different. I think its also best to keep the common devices in case of breaking changes upstream